store/unit_store.go

```diff
@@ -80,7 +81,10 @@ func (s unitStores) Defs(fs ...DefFilter) ([]*graph.Def, error) {
 	return allDefs, err
 }
 
-var c_unitStores_Refs_last_numUnitsQueried = 0
+var (
+	c_unitStores_Refs_last_numUnitsQueried = 0
+	c_mu                                   sync.Mutex
```

Comment:
Style. It's common to put the mutex as a "hat" on top of the things it protects. So you should swap these two variables.